### PR TITLE
support `landuse=civic_admin` and related `landuse`

### LINKED ISSUE #2985: area rendering for civic/governmental/administrative services
We should discuss how to render civic/civil/governmental/administrative/enforcement services, such as

- amenity=police/fire_station (fixes #848)
- amenity=embassy (mentioned in [#775](https://github.com/gravitystorm/openstreetmap-carto/issues/775#issuecomment-136331438))
- landuse for civic/governmental/administrative purposes, which is [proposed but undecided](https://wiki.openstreetmap.org/wiki/Proposed_features/Civic_admin) in tagging. 

In the discussion in #1624 some of them were included in "Societal amenities", however we are now using the societal yellow for kindergarten/school/university/college/hospital/community_centre/social_facility, which I consider a good grouping that should not be overloaden with more meanings. All these yellow areas are facilities that have in common to provide somehow direct care and service for their users.  kocio-pl's arts_centres would still fit in this group. Thereby I close #2087.

Thus we might think of a different colour group for the facilities listed in this issue. 
It might reuse commercial colour or be something new.

### PR DESCRIPTION
**Pull request by alexlancaster**
This PR supports the [`landuse=civic_admin`](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dcivic_admin) tag by rendering using a light blue background using the same styling as the [Tracestack Topo layer](https://www.openstreetmap.org/#map=18/42.36227/-71.06246&layers=P)

Fixes #2985

Changes proposed in this pull request:
- adds a light blue background with a thin border at higher zoom levels (z>17)
- supports the same colour for `landuse=` for `civic_admin` (an in-use tag) as well as `institutional`, `government` and `governmental`

Test rendering using local Kosmtik docker container:

* Boston Government Center: https://www.openstreetmap.org/#map=18/42.36249/-71.06250

  * Before (z18)

  OpenStreetMapCarto-Standard-z18

  * After (z18)
  
  OpenStreetMapCarto-Kosmtik-z18

  * After (z16)
  
  OpenStreetMapCarto-Kosmtik-z16

  * After (z19)

   OpenStreetMapCarto-Kosmtik-z19

### CODE REVIEW

**Comment by imagico** [comment]
Thanks for the pull request.

This change suggests to render the following tags in a unified design with a semi-transparent blue-green fill and a matching thick outline, similar to the ones used for marinas/theme parks but rendered in the landcover layer:

* `landuse=civic_admin`
* `landuse=institutional`
* `landuse=government`
* `landuse=governmental`

I see a number of issues with that:

The tags suggested to be rendered are IMO not suitable to be rendered in this style at this time.  Specifically:

* `landuse=civic_admin` has <900 uses, very sparse documentation
* `landuse=institutional` has 1667 uses, is clearly indicated to be a disputed tag and is based on the very vague English language term 'institution' without a clear language independent definition
* `landuse=government` has 516 uses and is undocumented
* `landuse=governmental` has 505 uses, only has a proposal page on the wiki and is almost exclusively used in China

All four tags massively overlap in their meaning and we have the general principle not to introduce new rendering of synonyms in this style.  As the documentation of [landuse=commercial](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dcommercial) correctly states, mappers widely use this tag also for government/administrative areas.  There is no consensus that using any of the above tags instead is a more desirable way of tagging.

In terms of rendering it is also significant to note that `office=government` (263k uses) - with a further differentiation tag `government=*` (115k uses) - is the more widespread (and more broadly accepted) way to map specifics about administrative infrastructure.  Rendering such in a differentiated form would be more significant for us.  Also keep in mind that much administrative infrastructure is building-only with no area outside the building being part of a specific land use.  Such cases would not show up with landuse rendering and such rendering could incentivize mappers to draw arbitrary wrapper polygons for the visual effect.

Regarding the design proposed - independent of the tagging this has the following issues:

* we don't want to use transparency in polygon fill colors because it leads to arbitrary and confusing color mixing with other colors.
* use of blue-green fill colors for something not vegetation or water related is problematic in terms of color systematics.  All our generic urban landuse colors are in the range of weak red-purple-violet tones and were carefully designed in perceptual color space to form a harmonic set (see #599).  Adding to that with an unrelated color is very confusing for the map user.
* heavy outline rendering in a semi-transparent blue is problematic because
  - depending on the background it can be confused with various other line signature, in case of blue color in particular with waterways.
  - it indicates a meaningful boundary where practically there is often none.
  - it hides actual mapping of physical boundaries (wall, fence etc.)
  - it looks weird in cases when urban landuse is split in mapping (which is widely considered a perfectly correct form of mapping for urban landuse)
  - rendering it in the landcover layer leads to it being covered by almost everything else - including other landuse fills, which leads to confusingly interrrupted line signatures.
  - overall this is a very heavy way of rendering polygons, which is not very suitable for a feature-rich style like ours.
* our systen of generic urban landuse rendering is already at the edge of being too complex to be intuitively understandable.  Hence if we'd introduce rendering of additional tags we would need to aggregate them further (like combining them with existing `landuse=commercial`).  This is of course only feasible if they have a well defined meaning and are well delineated from other tags in the first place.
 
Bottom line: I don't think any of the suggested tags currently qualifies for being rendered here and especially not all of them together as synonyms.  And if, at one point in the future, this changes we would need to choose a design that harmonically integrates with our existing urban landuse rendering.

Side note:  The rendering sample does not seem to match the current mapping - the polygon with the landuse tagging is this: https://www.openstreetmap.org/way/814380638 and i don't see the origin of the inner yard outlining shown.

**Comment by imagico** [comment]
Given the lack of arguments in favor of this change i am going to close this PR.  Please don't feel discouraged by that.  Your initiative was highly appreciated, just that the specific change, for the reasons given, is not considered suitable for this style.

**Comment by alexlancaster** [comment]
> Given the lack of arguments in favor of this change i am going to close this PR. Please don't feel discouraged by that. Your initiative was highly appreciated, just that the specific change, for the reasons given, is not considered suitable for this style.

I am planning to rework PR this and take into account the above feedback.  Should I open up another PR for that?

**Comment by imagico** [comment]
Yes, please do so.  But before investing substantial time into development it might be a good idea to open an issue on what you think needs addressing in light of what i mentioned.  Most PRs are made to address an issue where the problem to solve is clearly identified and where people already had the opportunity to comment and make suggestions.  It is fine to just open a PR right away of course - but often not the best approach.
